adafruit_ads1x15/ads1x15.py

```diff
@@ -131,6 +141,25 @@ def gains(self) -> List[float]:
         g.sort()
         return g
 
+    @property
+    def compqueue(self) -> int:
```

Comment:
Would this be a better description?

```suggestion
    def comparator_queue_length(self) -> int:
```

Reply:
Agreed.